test/jasmine/assets/mouse_event.js

```diff
@@ -1,6 +1,11 @@
-module.exports = function(type, x, y, opts) {
+module.exports = function(type, x, y) {
+    var options = {
+        bubbles: true,
+        clientX: x,
+        clientY: y
```

Comment:
@mdtusz we need this in order to propagate the click coordinates through the clicked layer.